clients/include/solver/testing_getrs_strided_batched.hpp

```diff
@@ -89,10 +91,13 @@ void setup_getrs_strided_batched_testing(const Arguments&                arg,
         }
     }
 
+    for(int i = 0; i < Ipiv_size; i++)
+        hIpiv32[i] = hIpiv64[i];
+
     // Copy data from CPU to device
     CHECK_HIP_ERROR(dA.transfer_from(hA));
     CHECK_HIP_ERROR(dB.transfer_from(hB));
-    CHECK_HIP_ERROR(hipMemcpy(dIpiv, hIpiv, Ipiv_size * sizeof(int), hipMemcpyHostToDevice));
+    CHECK_HIP_ERROR(hipMemcpy(dIpiv, hIpiv32, Ipiv_size * sizeof(int), hipMemcpyHostToDevice));
```

Comment:
I know here it is `hIpiv32` is that same convention going to followed in all the other functions as well? To differentiate between `hIpiv32` and `hIpiv64`

Reply:
Long-term we could probably just use 64-bit here and wouldn't need the duplicates I think. Just made as minimal changes here and will add the full support at a later point.